version/version_test.go

```diff
@@ -42,103 +40,6 @@ go version go1.14 linux/amd64`
 	require.Equal(t, want, info.String())
 }
 
-func TestCLI(t *testing.T) {
```

Comment:
May be we can some system test or alternative for this? We might need this test.

Reply:
There is a unit test just down that tests it.

Reply:
Oh, I missed it. Cool.